profile/src/test/resources/test-profiles/profile-test-simple-datetimes-errors.json

```diff
@@ -1,5 +1,5 @@
 {
```

Comment:
I'm not sure why this file (or the one below) isn't in the valid or invalid folders?